server/data/assessmentClient.test.ts

```diff
@@ -0,0 +1,68 @@
+import AssessmentClient from './assessmentClient'
+import { assessmentFactory } from '../testutils/factories'
+import paths from '../paths/api'
+import describeClient from '../testutils/describeClient'
+
+describeClient('AssessmentClient', provider => {
+  let assessmentClient: AssessmentClient
+
+  const token = 'token-1'
+
+  beforeEach(() => {
+    assessmentClient = new AssessmentClient(token)
+  })
+
+  describe('find', () => {
```

Comment:
When I started this work, I added functionality for finding an assessment, which later I realised I didn't need. I've kept it in as if we require this down the line the work is done, but let me know if you think this shouldn't be included.

Reply:
Hmm, we've got the GET endpoint on the API - do we not need it to allow people to view (and then update) an assessment? I suppose it doesn't hurt to have it just in case!

Reply:
Ah is it because we read it off the `Application` instead?

Reply:
Yes, exactly, we require the application for both `show` and `update` controller actions, so we don't need to find an assessment separately. I'll keep it in 👍🏾